content/docs/user-guide/data-management/track-sync-data.md

```diff
@@ -0,0 +1,204 @@
+
+<details>
+
+### Click to learn about _data codification_
+
+<abbr>DVC projects</abbr> separate data from code by replacing large files, ML
+models, etc. in your <abbr>workspace</abbr> with small [metafiles]; We call this
+strategy _codification_ (of the data). The actual file contents are cached in an
+independent data store and [linked] to your project.
+
+![Code vs. data](/img/code-vs-data.png) _Separating code from data_
+
+<admon type="info">
+
+In order to [avoid duplicate content][linked], and to support [versioning
+features][data versioning], files and directories are reorganized in the cache
+into a [content-addressable structure].
+
+</admon>
+
+[metafiles]: /doc/user-guide/project-structure
+[linked]: /doc/user-guide/data-management/large-dataset-optimization
+[content-addressable structure]:
+  /doc/user-guide/project-structure/internal-files#structure-of-the-cache-directory
+
+</details>
+
+To capture changes to tracked data, `dvc add` it again (alternatively,
+`dvc commit` will also do the trick). This caches the latest data present in the
+workspace and updates `.dvc` files accordingly (changes visible to Git).
+
+If you need to move or rename tracked data, use `dvc move`. To stop tracking
+data altogether, use `dvc remove`. To also remove it from the cache (either the
+latest or historic versions), use `dvc gc`. See [more details].
+
+Putting it all together, we can get an overview of the data in a project with
+`dvc data status`. This will list changes to DVC-tracked data as well as files
+unknown to DVC (or Git):
+
+```cli
+$ dvc data status
+Not in cache:
+        tmp/
+
+DVC committed changes:
+        added: data.xml
+        modified: data/features/
+
+DVC uncommitted changes:
+        deleted: model.pkl
+```
+
+<admon type="tip">
+
+Other commands partially related to tracking data: `dvc status`,
+`dvc unprotect`, `dvc list`, `dvc import`, `dvc import-url`.
+
+</admon>
+
+[similar to git]:
+  https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository
+[links it]: /doc/user-guide/data-management/large-dataset-optimization
+[more details]: /doc/user-guide/how-to/stop-tracking-data
+
+## Synchronizing data
+
+DVC lets you [codify your data], configure the project's storage location(s),
+and stop worrying about low-level operations like copying, moving, renaming,
+uploading and downloading, etc.
+
+At a minimum, you'll have two locations: your <abbr>workspace</abbr> and a
+<abbr>cache</abbr>. The [data tracking](#tracking-data) operations already keep
+these in sync most of the time.
+
+<admon type="tip">
+
+The `dvc commit` and `dvc checkout` plumbing commands let you force-sync them.
+This can be useful when unexpected errors occur (e.g. cache corruption).
+
+</admon>
+
+To enable additional storage locations, you can configure [DVC remotes] with the
+`dvc remote add` and `dvc remote modify` commands (see `dvc remote` for more
+options). Once this is done, use the `dvc push` and `dvc pull` commands to
```

Comment:
> you can configure [DVC remotes] with the `dvc remote add` and `dvc remote modify` commands (see `dvc remote` for more options).

- [ ] After https://github.com/iterative/dvc.org/pull/4058, replace for link to /doc/user-guide/data-management/remote-storage